### PR TITLE
Edit docs specification on Azure DevOps integration

### PR DESCRIPTION
**Pull request by jpitapeva**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/integrations/azure/discovery.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ The parameters available are:
 
 - **`host:`** _(optional)_ Leave empty for Cloud hosted, otherwise set to your self-hosted instance host.
 - **`organization:`** Your Organization slug (or Collection for on-premise users). Required.
-- **`project:`** _(optional)_ Your project slug. Wildcards are supported as show on the examples above. If not set, all projects will be searched. For a project name containing spaces, use both single and double quotes as in `project: '"My Project Name"'`.
+- **`project:`** _(required)_ Your project slug. Wildcards are supported as show on the examples above. If set '*' will search all projects. For a project name containing spaces, use both single and double quotes as in `project: '"My Project Name"'`.
```
Comment (awanlin) [inline docs/integrations/azure/discovery.md:70]: Can we add an "n" to the end of "show"? So it should read: "Wildcards are supported as shown on the examples above."

Also, this line "If set '*' will search all projects." would be better without the "If". How about: "Using a wildcard '*" will search all projects."

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for taking this on @jpitapeva, just left some changes to make this better

File: docs/integrations/azure/discovery.md (hunk 1 above)
Comment (jpitapeva) [reply docs/integrations/azure/discovery.md]: @awanlin i fixed :)

**Review by awanlin — APPROVED** [review]
Thank again @jpitapeva, this looks great and will reduce confusion 🚀 

Just need to wait for @Rugvip to do a quick review now

**Review by Rugvip — APPROVED** [review]
Thank you! 👍 

Threw prettier on it too 😁

**Pull request merged**